src/test/resources/pages/newlyJoinedInstructorFeedbackResultsPage.html

```diff
@@ -53,16 +53,24 @@
           <div class="col-sm-2">
               
               <div class="form-group">
-                  <form method="post" action="/page/instructorFeedbackResultsDownload">
-                  <div id="feedbackDataButtons">
-                      <input id="button_download" class="btn btn-primary" name="fruploaddownloadbtn" value="Download results" type="submit">
+                  <div class="col-sm-12">
+                      <form method="post" action="/page/instructorFeedbackResultsDownload">
+                      <div id="feedbackDataButtons">
+                          <input id="button_download" class="btn btn-primary btn-block" name="fruploaddownloadbtn" value="Download results" type="submit">
+                      </div>
+                      <input name="user" value="{$test.instructor}" type="hidden">
+                      <input name="fsname" value="Second team feedback session" type="hidden">
+                      <input name="courseid" value="AHPUiT.instr1.gma-demo" type="hidden">
+                      <input name="sectionname" value="All" type="hidden">
+                      
+                      </form><br>
+                      
+                      <div>
+                      <a class="btn btn-primary btn-block btn-tm-actions session-unpublish-for-test" data-original-title="Make responses no longer visible" data-placement="top" data-toggle="tooltip" href="/page/instructorFeedbackUnpublish?courseid=AHPUiT.instr1.gma-demo&amp;fsname=Second+team+feedback+session&amp;next=%2Fpage%2FinstructorFeedbacksPage%3Fuser%3Dinstructor&amp;user=instructor" onclick="return toggleUnpublishEvaluation('Second team feedback session');" title="">
```

Comment:
I am not able to pass all tests, the "instructor" user in this href should be {$test.instructor} instead.